Layout dialect fix

MarekB01 opened the pull request:
https://github.com/BroadleafCommerce/QA/issues/4960

Thymeleaf layout was upgraded in version 6.3 but on 6.2 we used old import. Now it is correct

MarekB01 committed:
PresentationLayer/QA-4960_layout_dialect_fix

Thymeleaf layout was upgraded in version 6.3 but on 6.2 we used old import. Now it is correct

oleksiimiroshnyk commented on broadleaf-thymeleaf3-presentation/src/main/java/org/broadleafcommerce/presentation/thymeleaf3/config/AbstractThymeleaf3DialectConfig.java:
```diff
@@ -80,7 +80,7 @@ public Set<IProcessor> blDialectProcessors() {
 
     @Configuration
-    @ConditionalOnClass(name = "nz.net.ultraq.thymeleaf.LayoutDialect")
+    @ConditionalOnClass(name = "nz.net.ultraq.thymeleaf.layoutdialect.LayoutDialect")
```
I would think it is better to use value field of ConditionalOnClass annotation rather than name - in this case if in the future class/package will change again build will fail as import will reference a non existing class

MarekB01 committed:
PresentationLayer/QA-4960_layout_dialect_fix

Thymeleaf layout was upgraded in version 6.3 but on 6.2 we used old import. Used class value if there will be updates in future the @ConditionalOnClass will always see the right class
